CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ project(${BOARD_LIB_NAME}
         LANGUAGES CXX C
         )
 
-add_library(${PROJECT_NAME} STATIC)
+add_library(${PROJECT_NAME} STATIC src/TMS/dev/TMP117I2CDevice.cpp include/TMS/dev/TMP117I2CDevice.hpp)
```

Comment:
These shouldn't be added here. CLion does this by default when you create a class, but these should instead be put below in the `target_sources` section